### PR TITLE
Completing Global Search

### PR DESCRIPTION
**Pull request by ogwuazors**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: src/pages/search/components/cards/siteCard.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        </Grid>
+      </>
+    );
+  }
+}
+
+export default withStyles(styles, { withTheme: true })(SiteCard);
+
+// const SiteCard = ({ data, classes, index }) => {
+//   const properties = [
```
Comment (iksheth) [inline src/pages/search/components/cards/siteCard.js]: @ogwuazors Can we remove commented section if not needed in future ?

File: src/pages/fileCentricCart/cartController.js [hunk 2]
```diff
@@ -42,6 +91,5 @@ const cartController = () => {
         }
     />
   );
-};
 
-export default cartController;
+export default cartController; */
```
Comment (iksheth) [inline src/pages/fileCentricCart/cartController.js]: @ogwuazors Why we are not exporting cartController this way?

**Review by iksheth — CHANGES_REQUESTED** [review]

File: src/pages/search/components/cards/siteCard.js (hunk 1 above)
Comment (ogwuazors) [reply src/pages/search/components/cards/siteCard.js]: It's been removed

**Comment by iksheth** [comment]
Closing this one as code base is re-written from Bento 3.7.0

**Pull request closed without merging**